.travis.yml

```diff
@@ -1,13 +1,13 @@
-dist: xenial
+dist: jammy
 bundler_args: --jobs=3 --retry=3
 os: linux
 cache: bundler
 language: ruby
 rvm:
-  - 2.7.6
   - 3.0.4
   - 3.1.2
   - 3.2.2
+  - 3.3.3
 before_install:
   - bundle config set deployment 'true'
 branches:
```

Comment:
Can you also switch this to be main instead of master when you switch the branch you are merging into?